view.php

```diff
@@ -184,12 +184,14 @@
 $table->data[] = array($strautorec, get_string('auto_rec_' . $zoom->auto_recording, 'mod_zoom'));
 
 $records = get_zoom_meeting_recordings($zoom->meeting_id);
```

Comment:
@nayanalinkstreet why not implement order by in the query in the method get_zoom_meeting_recordings itself?
Also try avoiding the query here in this file directly.